<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TankInfoWidget</class>
 <widget class="QWidget" name="TankInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>555</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>555</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>150</width>
    <height>555</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <widget class="QLabel" name="towerDown">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>322</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-green.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="towerUp">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>258</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-green.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="moveUp">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>138</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-green.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="moveLeft">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>170</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-green.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="fireButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-blue.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>238</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Tower</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="towerRight">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>290</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-green.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="moveRight">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>170</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-green.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="towerLeft">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>290</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-green.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="moveDown">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>202</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-green.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="playerName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;No Player&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLCDNumber" name="pointsLcdNumber">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(67, 67, 60);
color: rgb(90, 230, 10);</string>
   </property>
   <property name="digitCount">
    <number>3</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="laserLcdNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(67, 67, 60);
color: rgb(90, 230, 10);</string>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="laserAmmoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Ammo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="laserActive">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>380</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-blue.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Score</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="smokeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Smoke</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLCDNumber" name="smokeLcd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(67, 67, 60);
color: rgb(90, 230, 10);</string>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="SmokeActive">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-blue.png</pixmap>
   </property>
  </widget>
  <widget class="QLCDNumber" name="redeemerTimer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>521</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(67, 67, 60);
color: rgb(90, 230, 10);</string>
   </property>
   <property name="digitCount">
    <number>2</number>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="redeemerActive">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>521</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-blue.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="redeemerAvaliable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>521</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-green.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="smokeAvailiable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/inactive-green.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="smokeHasLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Has</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="smokeActiveLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Active</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="redeemerHasLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>501</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Has</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="redeemerActiveLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>501</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Active</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="laserActiveLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Active</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="redeemerTimerlabel">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>501</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Timer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="smokeTimerLabel">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>430</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Timer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="laserLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Laser</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="redeemerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Redeemer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
